src/containers/Tenant/TenantNavigation/TenantNavigation.tsx

```diff
@@ -31,16 +34,14 @@ export const TenantNavigation = () => {
         nextItem?.onForward();
     };
 
-    const getCurrentItem = () => navigationItems.find((item) => item.current) || navigationItems[0];
-
     return (
         <div className={b()}>
             <RadioButton
                 width="auto"
                 onUpdate={handleUpdate}
                 size="l"
                 className={b('body')}
-                defaultValue={getCurrentItem().id}
+                value={tenantPage}
```

Comment:
It seems that it will be enough just change property from `defaultValue` to `value`. 
Lets stick to this and return all tabs calculations to `useTenantNavigation`?

Reply:
I don't understand, why we need to set `current` property to `items` and get current item with `getCurrentItem`, if we could just use `tenantPage`. For me it just adds complexity